### ISSUE TITLE
Update import_workers.py to allow importing Dispatchers as well

### ISSUE THREAD

**Issue by jmdye** [Esri/workforce-scripts#16]
Would be useful if ```import_workers.py``` could also be used to add Dispatchers as well. I'd suggest changing the name of the script to ```import_users.py``` (breaking change?) and adding a userRole field where the value could be either ```worker``` or ```dispatcher```. 

Maybe this is already possible and I just missed the documentation on it

**Comment by crai3162** [comment]
Hi @jmdye, this is something we discussed, but decided not to go in that direction as Workers is primary case where importing is needed due to the number of workers.

Are you asking for a script to import dispatchers for completeness?  Or, do you have a large number of dispatchers to add where performing this manually would cause an issue.

thanks for the feedback.

**Comment by jmdye** [comment]
Dozens of projects, different dispatchers for each. Easy but tedious to go into each project and manually add dispatchers. 

Lack of import_dispatchers capability also makes it impossible to build a soup-to-nuts workflow for creating new projects. For instance, if your assignmentsCSV uses the ```-dispatcherIdField```, you then have to pause before running ```create_assignments_from_csv``` so that you can go and manually add the dispatchers to the project to ensure that the dispatcherIdField is valid and corresponds to the correct dispatcher

**Comment by crai3162** [comment]
Hi @jmdye all good points, thanks for taking the time to add them.

We'll take a look at adding a script or updating the existing script to do this, for simplicity, we'll probably look at creating a new script specifically for Dispatchers.

**Comment by jmdye** [comment]
cool. whatever works best. thanks for considering it. One thing I'd call out - I think I saw some logic in one of the scripts to ensure workers are unique and aren't added to the project more than once. (maybe I was hallucinating, I don't recall specifically where I think I saw it). In any case, we usually allow our Dispatchers to also have the Worker role so that they can use the mobile app to see what the field worker sees. Also makes it easier if a Dispatcher needs to update something for the Worker. We're not on Win10 yet, so no Workforce Desktop here =(
